### PR TITLE
update variable for colors to theme

### PR DESCRIPTION
**Pull request by pangthebabydustbunny**

### CODE REVIEW

File: src/reusable/SortableDropdown.re [hunk 1]
```diff
@@ -12,7 +12,7 @@ module Styles = {
       boxShadow(
         Shadow.box(~x=`zero, ~y=`px(2), ~blur=`px(4), Css.rgba(0, 0, 0, `num(0.08))),
       ),
-      backgroundColor(Colors.white),
+      backgroundColor(rgb(255, 255, 255)),
```
Comment (evilpeach) [inline src/reusable/SortableDropdown.re:15]: Use `Theme.white` instead.

File: src/reusable/CssHelper.re [hunk 2]
```diff
@@ -76,7 +76,7 @@ let selectWrapper = (~size=14, ~pRight=16, ~pRightSm=pRight, ~mW=500, ~fontColor
 
 let infoContainer =
   style([
-    backgroundColor(Colors.white),
+    backgroundColor(rgba(255, 255, 255, `num(1.))),
```
Comment (evilpeach) [inline src/reusable/CssHelper.re:79]: Use `Theme.white`

**Review by evilpeach — CHANGES_REQUESTED** [review]
Let's talk to our designer about what we should replace `Colors.gray7` (#555555) by other color.

File: src/components/modal/connect/ConnectWithLedger.re [hunk 3]
```diff
@@ -19,12 +18,12 @@ module Styles = {
...
     ]);
 
-  let oval =
+  let oval = (theme: Theme.t) =>
     style([
       backgroundImage(
         `linearGradient((
           `deg(180.),
-          [(`percent(0.), Colors.blue2), (`percent(100.), Colors.bandBlue)],
+          [(`percent(0.), Css.hex("CED4FD")), (`percent(100.), theme.baseBlue)],
```
Comment (pzshine) [inline src/components/modal/connect/ConnectWithLedger.re:26]: Is it should `lightenBlue`?

**Comment by pzshine** [comment]
Please remove `src/.DS_Store`

File: src/components/UserAccount.re [hunk 4]
```diff
@@ -57,14 +57,15 @@ module ConnectBtn = {
 module DisconnectBtn = {
   [@react.component]
   let make = (~disconnect) => {
+    let ({ThemeContext.theme}, _) = React.useContext(ThemeContext.context);
```
Comment (evilpeach) [inline src/components/UserAccount.re:60]: It's no need to use context.

File: src/components/UserAccount.re [hunk 5]
```diff
@@ -57,14 +57,15 @@ module ConnectBtn = {
...
     <div
       className={Css.merge([
         CssHelper.flexBox(~justify=`center, ~align=`center, ()),
         CssHelper.clickable,
         Styles.disconnect,
       ])}
       onClick={_ => disconnect()}>
-      <Text value="Disconnect" weight=Text.Medium color=Colors.bandBlue nowrap=true block=true />
+      <Text value="Disconnect" weight=Text.Medium color={theme.baseBlue} nowrap=true block=true />
```
Comment (evilpeach) [inline src/components/UserAccount.re:68]: Just use `Theme.baseBlue`

File: src/components/UserAccount.re [hunk 6]
```diff
@@ -163,7 +164,7 @@ let make = () => {
           className={Css.merge([CssHelper.flexBox(), CssHelper.clickable])}
           onClick={_ => setShow(prev => !prev)}>
           <div className={Styles.oval(theme)}>
-            <Icon name="fal fa-user" color=Colors.white />
+            <Icon name="fal fa-user" color={theme.white} />
```
Comment (evilpeach) [inline src/components/UserAccount.re:167]: `Theme.white`

File: src/components/modal/connect/ConnectWithMnemonic.re [hunk 7]
```diff
@@ -21,14 +21,14 @@ module Styles = {
...
-  let mnemonicHelper =
+  let mnemonicHelper = (theme: Theme.t) =>
     style([
       width(`px(130)),
       height(`px(16)),
       display(`flex),
       justifyContent(`spaceBetween),
       alignContent(`center),
-      color(Css.hex("5269FF")),
+      color(theme.baseBlue),
```
Comment (evilpeach) [inline src/components/modal/connect/ConnectWithMnemonic.re:31]: Remove `mnemonicHelper` function cuz it didn't use.

File: src/components/modal/connect/ConnectWithLedger.re [hunk 8]
```diff
@@ -19,12 +18,12 @@ module Styles = {
       justifyContent(`spaceBetween),
     ]);
 
-  let oval =
+  let oval = (theme: Theme.t) =>
```
Comment (evilpeach) [inline src/components/modal/connect/ConnectWithLedger.re:21]: remove this function. It didn't use.
remove `rFlex` function as well.

File: src/components/modal/submitTx/SubmitTxModal.re [hunk 9]
```diff
@@ -36,7 +36,7 @@ module Styles = {
       background(rgba(255, 255, 255, `num(1.))),
       border(`px(0), `solid, hex("FFFFFF")),
       width(`px(135)),
-      focus([outlineColor(Colors.white)]),
+      focus([outlineColor(rgba(255, 255, 255, `num(1.)))]),
```
Comment (evilpeach) [inline src/components/modal/submitTx/SubmitTxModal.re:39]: Theme.white?

File: src/reusable/CTooltip.re [hunk 10]
```diff
@@ -18,19 +18,18 @@ module Styles = {
...
       display(`block),
       fontSize(`px(fsize_)),
       textAlign(align_),
-      backgroundColor(Colors.gray7),
+      backgroundColor(`rgba((85, 85, 85, `num(1.0)))),
       borderRadius(`px(4)),
       maxWidth(`px(maxWidth_)),
       unsafe("width", "max-content"),
-      color(Colors.white),
+      color(theme.white),
```
Comment (evilpeach) [inline src/reusable/CTooltip.re:32]: Theme.white

File: src/reusable/CTooltip.re [hunk 11]
```diff
@@ -150,7 +149,7 @@ module Styles = {
             top(`percent(50.)),
             transform(`translateY(`percent(-50.))),
             left(`px(-10)),
-            borderRightColor(Colors.gray7),
+            borderRightColor(`rgba((85, 85, 85, `num(1.0)))),
```
Comment (evilpeach) [inline src/reusable/CTooltip.re:152]: @pzshine Should we use theme or use `rgba`?

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: src/reusable/CTooltip.re (hunk 11 above)
Comment (pangthebabydustbunny) [reply src/reusable/CTooltip.re:152]: @evilpeach  P' @pzshine recommended to use hex instead ka since there's no similar color in theme

**Review by evilpeach — APPROVED** [review]

**Pull request merged**